src/Parser/Core/Modules/AlwaysBeCasting.js

```diff
@@ -172,40 +172,41 @@ class AlwaysBeCasting extends Analyzer {
     downtime: '/img/afk.png',
   };
   statistic() {
-    if (!this.showStatistic) {
+    if (!this.showStatistic || this.owner.boss.fight.noDowntimeSuggestion === true) {
```

Comment:
Boss may be null/undefined in which case this will crash. Also checking for `=== true` can be omitted.